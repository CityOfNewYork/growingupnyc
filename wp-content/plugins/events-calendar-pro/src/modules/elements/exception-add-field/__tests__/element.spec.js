import React from 'react';
import renderer from 'react-test-renderer';
import ExceptionAddField from '../element';

describe( 'ExceptionAddField', () => {
	test( 'should match snapshot', () => {
		const component = renderer.create(
			<ExceptionAddField onClick={ jest.fn() } />
		);
		expect( component.toJSON() ).toMatchSnapshot();
	} );
} );
