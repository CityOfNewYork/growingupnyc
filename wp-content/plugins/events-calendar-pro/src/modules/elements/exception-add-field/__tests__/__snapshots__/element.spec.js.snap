// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExceptionAddField should match snapshot 1`] = `
<aside
  className="tribe-editor__events-pro__add-field"
>
  <button
    className="tribe-editor__events-pro__add-field__button"
    onClick={[MockFunction]}
    type="button"
  >
    <span
      className="tribe-editor__events-pro__add-field__button__plus"
    >
      +
    </span>
    <span>
      Add Exception
    </span>
  </button>
</aside>
`;
